.travis.yml

```diff
@@ -14,6 +14,7 @@ before_install:
   - go get -u github.com/golang/lint/golint
   - go get -u github.com/mattn/goveralls
   - go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
+  - go get -u github.com/stretchr/testify/assert
```

Comment:
I don't think this line should be needed. The above lines are installing tools to use, e.g. `golint`.

As far as I can tell, assert is a library, and this will be downloaded by `go get -t -v ./...` below, as it is an imported dependecy of the source code.

**Edit:** I'll merge the PR as is, and upload a change to remove this line and see if it works on Travis.